trimme_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'base_decorator'
+require_relative 'person'
+require_relative 'capitalize_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    super[0...10]
+  end
+end
```

Comment:
- [ ] The correct  name should have a maximum of 10 characters. If it's longer the method should trim the word.
  _Hint:_  ` 
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
`